app/controllers/registration/business/MandatoryTradingNameController.scala

```diff
@@ -62,7 +63,11 @@ class MandatoryTradingNameController @Inject()(val sessionService: SessionServic
               _ =>
                 vatRegistrationService.partyType.map {
                   case NETP => Redirect(controllers.registration.returns.routes.ZeroRatedSuppliesResolverController.resolve)
-                  case _ => Redirect(controllers.registration.business.routes.ImportsOrExportsController.show)
+                  case _ => if (isEnabled(OtherBusinessInvolvement)) {
```

Comment:
Is putting the redirect here and in trading name controller right? The story (and flow diagram) say that it should sit right after the sic and compliance block, there's 3 pages where this redirect would need to sit.